### ISSUE TITLE
Analytics KPI dashboard request for [Decision Review Team - Request a Board Appeal]

### ISSUE THREAD

**Issue by saderagsdale** [department-of-veterans-affairs/va.gov-team#55818]
# Request a New Standardized KPI Dashboard

## What this form is for

Use this ticket template to request a new standardized KPI dashboard on Domo for your product. To make updates to a product already on a standardized dashboard, please use the [Request an Update to your Standardized KPI Dashboard template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=michelle-dooley&labels=analytics-insights%2C+analytics-request%2C+kpi-dashboard&template=analytics-request-update-kpi-dashboard.md&title=Analytics+KPI+dashboard+update+request+for+%5BTeam%5D).

Standardized dashboards are currently available for Forms, Content, & Search products only. To see if you should request a standardized dashboard, check [Should I request a standardized KPI dashboard](https://docs.google.com/presentation/d/16mRcslbkWtW44rlsfQGXZnHtQ3GoLjn6GpFnd3Gvva8/edit#slide=id.p). If you’re not sure which dashboard(s) your product(s) qualify for, check the [Eligibility Dashboard](https://va-gov.domo.com/page/1550814739).

If your VA.gov product is not eligible for a standardized dashboard, you may be able to request an “edge case” dashboard. Please reach out to the Platform Analytics & Insights team on #vfs-platform-support for more information on this process.

### Expected delivery timelines

Once you’ve submitted this ticket template, your KPI dashboard request will be pulled into the Analytics team’s workflow the following Wednesday during Platform Sprint Planning or Refinement. We expect to deliver your completed dashboard within **2-4 business** days from this time. 

## Instructions for the VFS team
In order to pull the correct data for your product, we need your help to define what pages and events to include, in addition to basic product info. We’ll need a filled out [KPI request template](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988) before we can pull this ticket into our sprint.

*NOTE: If you do not have Google access, you can download an Excel version of the template [here](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/1586823275/KPI%20Dashboard%20Product%20Details%20Template%20-%20MAKE%20A%20COPY.xlsx?inst-v=ced1f4dd-e997-4aa2-9ae8-2cebe369b42f).*

- [x] Make a copy of the [Google Sheet](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988) [(or download Excel version)](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/1586823275/KPI%20Dashboard%20Product%20Details%20Template%20-%20MAKE%20A%20COPY.xlsx?inst-v=ced1f4dd-e997-4aa2-9ae8-2cebe369b42f)
- [x] Complete worksheet as instructed
- [x] Link your completed spreadsheet [here](https://docs.google.com/spreadsheets/d/1opinCLJ-m60bnnLZQ0NyHj-cbAEXmQRJT9sYt8kyJ8A/edit#gid=0)
- [x] Check your sharing settings to make sure the Analytics team will be able to access it
- [x] Add your team label to follow progress

**Additional Info**

- Please use the [KPI Framework](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/Analytics%20Playbook/va-gov-platform-analytics-kpi-framework.pdf) for more information on the metrics used in the Standard KPI dashboards. 
- If you need any analytics support, read through the [How to request Analytics support](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Analytics-customer-support-guidelines.1586823275.html#Analyticscustomersupportguidelines-4.RequestaKPIDashboardinDomo).
- If you do not have Domo access, please [request access here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=joanneesteban&labels=analytics-insights%2C+analytics-request%2C+access-request&template=analytics-request-google-analytics-domo-access.md&title=Request+access+to+Google+Analytics+and%2For+Domo).

**Comment by saderagsdale** [comment]
@eileen-coforma FYI this request is submitted

**Comment by michelle-dooley** [comment]
@saderagsdale - thanks for your submission. I will review with team and get back to you with any questions and timing.

**Comment by saderagsdale** [comment]
@michelle-dooley Hey Michelle! I believe the rollout for this one started before our onboarded, on 4/1/2022. Let's go as far back as we can.

**Comment by dorseypa1** [comment]
Can I have access to the Dashboard Product Details google sheet related to this request?  Thank you.

**Comment by eileen-coforma** [comment]
@dorseypa1 You should have access now

**Comment by dorseypa1** [comment]
I do have access now.  Thank you.

**Comment by saderagsdale** [comment]
@dorseypa1 Hey Phil, what's the ETA on this one?

**Comment by dorseypa1** [comment]
The coding is done and they are in QA testing right now.  I'll let Michelle comment on the ETA.  @michelle-dooley , please correct me if I am wrong.

**Comment by michelle-dooley** [comment]
Yep, working on QA.

**Comment by michelle-dooley** [comment]
@saderagsdale that data in Domo, you are welcome to do some of your own QA as well if you like and provide feedback.

**Comment by eileen-coforma** [comment]
@michelle-dooley I checked DOMO and saw that the start date for this form is in the wrong year. It should've started 4/2/2022, but the start date for the data is 4/2/2023.

**Comment by michelle-dooley** [comment]
@eileen-coforma - yes I see that we need to backfill that data.  Does everything else look good before we do that?

**Comment by eileen-coforma** [comment]
@michelle-dooley 
Can I get an ETA on the data backfill for NoD?

**Comment by michelle-dooley** [comment]
@eileen-coforma - @dorseypa1 is working on the backfill now, we will let you know once complete.

**Comment by michelle-dooley** [comment]
@eileen-coforma sorry for the delay on the backfill, we ran into some issues with the size of the table we are working through.

**Comment by eileen-coforma** [comment]
@michelle-dooley Is there an updated ETA on this?

**Comment by michelle-dooley** [comment]
@eileen-coforma the backfill has been completed.  I also looked into the previous page table and am seeing some urls that look to be form steps but they have suffixes that are not part of the funnel as defined in set up.

For example:
Form step 14 - www.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/area-of-disagreement](http://www.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/area-of-disagreement

But when I look at the urls in that table after form steps are removed I see those highlighted in yellow.  Do we need to create forms steps for each of those?

![image](https://user-images.githubusercontent.com/93597659/234095100-7ffcb923-4426-4e83-840e-dc802d73380d.png)

**Comment by eileen-coforma** [comment]
@michelle-dooley I think most of the data is correct now. There is one section that's still not reporting the right data, which is `Top pages viewed before accessing the form`. 
![Screenshot 2023-04-24 at 3 29 44 PM](https://user-images.githubusercontent.com/111457818/234096908-c2f75503-8c92-419e-bf04-cff16956da2a.png)

RE: your question above, they are not form steps. They are additional issues (ie. if a veteran chooses to add an issue manually), so it's not a required step within the form flow. We are still tracking them to see how many issues they are choosing to add, but I don't think it would affect the funnel.

**Comment by michelle-dooley** [comment]
@eileen-coforma - the issue is we remove/filter out form steps from this table as defined in set up (this is how we do it for all Forms that are part of this dashboard).  Because the steps I outlined above are not part of the funnel they are not getting filtered out (they are viewed same as any other previous page).  I cannot change the table schema behind this without affecting all other Forms.  I just did a manual filter for this card but it does slow it down a bit.
![image](https://user-images.githubusercontent.com/93597659/234298077-56672548-05ce-413b-a453-e9abf9a1d7ce.png)

**Comment by eileen-coforma** [comment]
@michelle-dooley I see, that makes sense. Thanks for filtering it out, the table makes more sense. Is the timeframe affected because of the delay? The data still says it's between "Apr 02, 2023 - Apr 22, 2023" instead of 2022.

**Comment by michelle-dooley** [comment]
@eileen-coforma I am not sure why the history isn't populating there since we have it.  I will have someone look into it.  I am going to close this ticket though since the work to add is complete.  I will let you know once that history is retrieved.  @dorseypa1 - this relates to my questions about the previous_page_path not populating history for this form.

**Comment by michelle-dooley** [comment]
I will create a bug ticket for this work so we do not lose sight.

**Comment by michelle-dooley** [comment]
@eileen-coforma - this update is complete.

**Comment by eileen-coforma** [comment]
Thank you!
